jib-core/src/test/java/com/google/cloud/tools/jib/http/WithServerConnectionTest.java

```diff
@@ -52,4 +56,22 @@ public void testErrorOnSecondSend() throws IOException, InterruptedException {
       }
     }
   }
+
+  @Test
+  public void testInsecureConnection()
+      throws IOException, InterruptedException, GeneralSecurityException, URISyntaxException {
+    try (TestWebServer server = new TestWebServer(true);
+        Connection connection =
+            new Connection.Builder(new URL(server.getEndpoint()))
```

Comment:
Generally, in this codebase, we've been doing the builder instantiation with a static method `builder()` so that this can be like `Connection.builder(new URL(server.getEndpoint())).doNot...`